swiftly/client/standardclient.py

```diff
@@ -514,19 +516,19 @@ def request(self, method, path, contents, headers, decode_json=False,
                         method, conn_path + path, contents, titled_headers)
                 else:
                     conn.putrequest(method, conn_path + path)
-                    content_length = None
```

Comment:
Gotcha on the None/int comparisons no longer being allowed. Should be able to work around that by going back to the original None or int code and changing the `left=content_length` line to `left=content_length or 0`.

As far as urllib/requests, I don't remember at this point. Probably some difficulty in unalterable behavior (probably around headers), or with large (5GB) requests, or something. Considering some code and ideas are going on four years old now, feel free to fork and try out other ideas if you want. Job changes mean I don't use this on a day-to-day basis anymore, so I admit things have probably languished.